#! /bin/sh /usr/share/dpatch/dpatch-run
## 02-dfl-error-dir.dpatch by  <mark@wikimedia.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Default error dir is under /usr/share/squid for Debian

@DPATCH@
diff -urNad squid-2.6.3~/src/defines.h squid-2.6.3/src/defines.h
--- squid-2.6.3~/src/defines.h	2006-09-18 17:00:23.000000000 +0000
+++ squid-2.6.3/src/defines.h	2006-09-18 17:01:01.000000000 +0000
@@ -249,7 +249,7 @@
 
 /* were to look for errors if config path fails */
 #ifndef DEFAULT_SQUID_ERROR_DIR
-#define DEFAULT_SQUID_ERROR_DIR "/usr/local/squid/etc/errors"
+#define DEFAULT_SQUID_ERROR_DIR "/usr/share/squid/errors/Wikimedia"
 #endif
 
 /* gb_type operations */
